external_deps/build.sh

```diff
@@ -889,6 +889,34 @@ build_naclports() {
 	cp pepper_*"/ports/lib/newlib_pnacl/Release/"{liblua.a,libfreetype.a,libpng16.a} "${PREFIX}/pnacl_deps/lib"
 }
 
+
+NATIVECLIENT_REVISION=27ccec9ff58b04b7635cd8a7b5b997a58c071952
+# Only builds nacl_loader and nacl_helper_bootstrap for now, not IRT.
+build_naclruntime() {
+	case "${PLATFORM}" in
+	linux-amd64-default)
+		local NACL_ARCH=x86-64
+		;;
+	*)
+		log error "naclruntime not implemented for ${PLATFORM}"
+		;;
+	esac
+
+	local dir_name="DaemonEngine-native_client-${NATIVECLIENT_REVISION:0:7}"
+	local archive_name="native_client-${NATIVECLIENT_REVISION}.zip"
+
+	download_extract naclruntime "${archive_name}" \
+		"https://api.github.com/repos/DaemonEngine/native_client/zipball/${NATIVECLIENT_REVISION}"
+
+	"${download_only}" && return
+
+	cd "${dir_name}"
+	scons "platform=${NACL_ARCH}" MODE=opt-host naclsdk_validate=0 sysinfo=0 sel_ldr
```

Comment:
`werror=0`

Reply:
Done. Apparently this warning only started working in Clang 17; I was using 14.